src/css/leftpanel.scss

```diff
@@ -533,12 +533,17 @@
 .layers-control-container {
   color: $dark-grey;
 
+  .buffer {
+    padding-bottom: 1rem;
+  }
+
```

Comment:
instead of creating this buffer div that adds padding can we remove it and add `margin-top: 1rem` to the `.layers-control-checkbox` class?